tests/upgrades/test_host.py

```diff
@@ -0,0 +1,152 @@
+"""Test Host/Provisioning related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Hosts
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from airgun.session import Session
+from fauxfactory import gen_string
+from nailgun import entities
+from robottelo.api.utils import skip_yum_update_during_provisioning
+from robottelo.constants import FOREMAN_PROVIDERS
+from robottelo.helpers import download_gce_cert
+from robottelo.test import APITestCase, settings
+from upgrade_tests import pre_upgrade, post_upgrade
+from upgrade_tests.helpers.scenarios import create_dict, get_entity_data
+from wait_for import wait_for
+
+
+GCE_SETTINGS = dict(
+    project_id=settings.gce.project_id,
+    client_email=settings.gce.client_email,
+    cert_path=settings.gce.cert_path,
+    zone=settings.gce.zone,
+    cert_url=settings.gce.cert_url,
+)
+
+
+class scenario_positive_gce_host_compute_resource(APITestCase):
+    """The host can be provisioned on GCE CR created in previous version
+
+    :steps:
+
+        1. In Preupgrade Satellite, create GCE Compute Resource
+        2. Upgrade the satellite to next/latest version
+        3. Postupgrade, The Compute Resource attributes can be manipulated
+        4. The host can be provisioned on GCE CR created in previous satellite version
+
+    :expectedresults:
+
+        1. The host should be provisioned on GCE CR created in previous version
+        2. The GCE CR attributes should be manipulated
+    """
+    @classmethod
+    def setUpClass(cls):
+        cls.hostname = None
+        cls.arch = entities.Architecture().search(query={'search': 'name=x86_64'})[0]
+        cls.img_uuid = '7726764279310511390'
```

Comment:
Is this random value or defined somewhere?

Reply:
This is a UUID of the image on Google Cloud. This is not going to change.

Reply:
I would suggest to keep it in constant with docstring added about how and where to find this value for future reference.